packages/verification-core/src/pdf/PdfService.js

```diff
@@ -0,0 +1,53 @@
+import Bowser from 'bowser'
+import PdfWasm from './pdf.wasm.init'
+import PdfWorker from 'web-worker:./pdf.worker'
+
+export default class PdfService {
+    constructor(pdfWasmUrl) {
+        this.pdfWasmUrl = new URL(pdfWasmUrl, location.origin)
+
+        const browser = Bowser.getParser(window.navigator.userAgent)
+        this.isNonChromiumEdge = browser.satisfies({ edge: '<=18' })
```

Comment:
Maybe avoid negation in the name to increase code readability.

Reply:
It's not really a negation, because "is non chromium edge" is similar to "is firefox"